CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Prefix the change with one of these keywords:
 ## Unreleased
 
 - Added: Checkbox has a new variant: 'tertiary'
+- Removed: css prop for SearchBar and SearchBarToggle.
```

Comment:
Is this a breaking change when the `css` prop is used to style these components?

Reply:
it is. marked as breaking now